requirements/requirements-dev.txt

```diff
@@ -15,5 +15,5 @@ sphinx
 sphinx-rtd-theme
 tensorboard
 torchvision
-transformers>=4.32.1
+transformers>=4.38.2
```

Comment:
Last release that did not contain this change was 4.38.2, 4.39.dev0 looks to have this, but want to update this to the next official release.